website_sale_qty/static/src/js/website_sale_qty.js

```diff
@@ -0,0 +1,47 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('website_sale_qty', function(require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    base.ready().done(function() {
+        var radios = $('.js-pqt-input');
+
+        if (radios.length > 0) {
+            var radio_map = {};
+            radios.each(function(index, element) {
```

Comment:
`_` is already included in Odoo, so you should use it. You will need to `var _ = require('_');` though